tracer/src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.csproj

```diff
@@ -157,6 +157,7 @@
     <PackageReference Include="NETStandard.Library" Version="2.0.3" />
     <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />
     <PackageReference Include="System.Management" Version="5.0.0" />
+    <PackageReference Include="System.Security.Principal.Windows" Version="5.0.0" />
```

Comment:
Is this one really necessary? I wonder if it was added accidentally? 🤔

Reply:
It is not required. I have updated the code. Thanks!